indico_zodbimport/modules/event_categories.py

```diff
@@ -44,7 +44,7 @@ def migrate_event_categories(self):
         for conf in committing_iterator(self._iter_events()):
             try:
                 category_chain = self.category_mapping[int(conf._Conference__owners[0].id)]
-            except KeyError:
+            except IndexError:
```

Comment:
I'd catch both. At least in the old indico/cern DB there were events with an invalid parent category